### PR TITLE
fix the post install job script when validatingwebhookconfiguration is absent

### PR DESCRIPTION
**Pull request by ayj**

### CODE REVIEW

**Comment by ayj** [comment]
cc @costinm

File: install/kubernetes/helm/istio/templates/install-custom-resources.sh.tpl [hunk 1]
```diff
@@ -13,13 +14,16 @@ pathToResourceYAML=${1}
...
-        kubectl -n {{ .Release.Namespace }} get deployment istio-galley 2>/dev/null
+        /kubectl -n {{ .Release.Namespace }} get deployment istio-galley 2>/dev/null
         if [ "$?" -eq 0 ]; then
             break
         fi
         sleep 1
     done
     /kubectl -n {{ .Release.Namespace }} rollout status deployment istio-galley
+    if [ "$?" -ne 0 ]; then
+        exit 1
```
Comment (gyliu513) [inline install/kubernetes/helm/istio/templates/install-custom-resources.sh.tpl]: can you add a log message here tell end user that the script failed?

**Comment by ayj** [comment]
/retest 

```
W0629 00:27:16.316] + go test -v istio.io/istio/security/tests/integration/certificateRotationTest --tag 16562383859e1123513c2f8c0511364f6614151e --hub gcr.io/istio-testing -kube-config=/home/bootstrap/.kube/config
I0629 00:27:51.305] 2018-06-29T00:27:51.303800Z	error	failed to create test namespace: failed to create a namespace (error: Post https://35.199.19.58/api/v1/namespaces: dial tcp 35.199.19.58:443: i/o timeout)
I0629 00:27:51.306] 2018-06-29T00:27:51.303834Z	error	test environment creation failure
I0629 00:27:51.306] FAIL	istio.io/istio/security/tests/integration/certificateRotationTest	31.055s
```

**Pull request merged**